README.md

```diff
@@ -101,18 +101,32 @@ Be aware that Cloud Shell has [several limitations][cloud-shell-limitations],
 in particular an inactivity timeout that will close running shells after 20
 minutes. Please consider it only for blueprints that are quickly deployed.
 
-## Blueprint Warnings and Errors
+## Blueprint Validation
```

Comment:
The `Enable GCP APIs` probably needs a minor update with the addition of this feature.

Reply:
There is a task to handle the docs on cloud.google.com

Reply:
nit: My thought was that "These will be caught when you perform `terraform apply` but you can
save time by enabling them upfront." on line 114 might say something about create instead of apply. But not critical.